### DISCUSSION TITLE
How to increase the maximum of cells

### DISCUSSION THREAD

**Discussion by TimoRuetten** [exceljs/exceljs#2516] [Q&A]
Is there any possible way to increase the maximum of cells ? Currently I get this error:

Error: Out of bounds. Invalid column letter: _

You can reproduce this error when you add more cells then 26. When removing the 27th cell the error does not appear.

``` javascript
cells: [
  {
    content: '1st cell A',
      style: {/*...*/}
  },
  /* cells 2-26, B-Z*/
  {
      content: '27th cell AA',
      style: {/*...*/}
  }
]
```

**Edit**  The error I told you is just coming when you add a "style" property to the cells after cell Z. When you do not add a "style" property the error will not show up but the content will also not be shown in the created file.

**Comment by TimoRuetten** [comment]
@guyonroche Is there a way to get a response by you or are you not developing anymore on this project ?

**Comment by MeisterBerg** [comment]
I have the same issue here. Need more cells than A-Z. Is that possible?
